### PR TITLE
#9 edit Entities and Migrations

### PR DESCRIPTION
**Pull request by leyla-rezaei**

### CODE REVIEW

File: JobSeek/JobSeek.Api/Migrations/20230221195509_add-edit-make-parent-jobcategory-without-parent.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```
Comment (zoha-shobbar) [inline JobSeek/JobSeek.Api/Migrations/20230221195509_add-edit-make-parent-jobcategory-without-parent.cs]: remove this empty migration

File: JobSeek/JobSeek.Api/appsettings.json [hunk 2]
```diff
@@ -7,6 +7,6 @@
   },
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "DefaultConnection": "Data Source=ZOHAPC\\ZSHO_SRVR19_01;Initial Catalog=JobSeek;Integrated Security=True;Pooling=False;Encrypt=False"
```
Comment (zoha-shobbar) [inline JobSeek/JobSeek.Api/appsettings.json:10]: dont remove this line, just comment it

**Review by zoha-shobbar — COMMENTED** [review]
resolve comments

**Comment by leyla-rezaei** [comment]
#9 this closes

**Review by zoha-shobbar — APPROVED** [review]

**Pull request merged**